public/index.html

```diff
@@ -44,6 +44,6 @@ <h1 class="control__title">TASKMANAGER</h1>
 
       <!-- Контент -->
     </main>
-    <script src="../src/main.js"></script>
+    <script src="./bundle.js"></script>
```

Comment:
то же здесь, `./` не нужен
```suggestion
    <script src="bundle.js"></script>
```